index.bs

```diff
@@ -554,26 +554,26 @@ spec: XHR; urlPrefix: https://xhr.spec.whatwg.org/
   # Impact on Web Platform # {#impact}
 
   Content inside a suborigin namespace is restricted in the same way that other
-  origins are restricted. There are some additional restrictions as well, in order
-  to simplfy some complicated cases, and there are also some loosening of
-  same-origin restrctions in order to facilitate and ease adoption of suborigins
-  for developers.
+  origins are restricted. There are some additional restrictions as well, in
+  order to simplify some complicated cases.
 
-  ## Relationship with Sensitive Permissions ## {#sensitive-permissions}
+  ## Storage ## {#storage}
 
-  User agents MUST prevent a page running in a suborigin from accessing stateful
-  mechanisms (e.g., localStorage, sessionStorage, document.cookie) tied to the
-  parent, physical origin. Instead, the user agent SHOULD create a new object
-  tied to the namespaced suborigin.
+  The storage APIs, such as <a>localStorage</a> and <a>sessionStorage</a>, are
```

Comment:
So one of the advantages of the SHOULD wording was that if a UA finds it too painful to implement sessionStorage/localStorage, it doesn't have to. The new wording is a bit more unclear on this. wdyt?